### PR TITLE
Update README.mdx

### PR DESCRIPTION
**Pull request by cowmarketing**
Updates Readme with first page info on MEV Blocker documentation

### CODE REVIEW

File: docs/mevblocker/README.mdx [hunk 1]
```diff
@@ -1,4 +1,40 @@
----
-title: MEVBlocker
-draft: true
----
+# MEV Blocker
+
+A product developed and stewarded by CoW DAO, Beaver Builder and Gnosis DAO.
+
+IMG
```
Comment (mfw78) [inline docs/mevblocker/README.mdx:5]: Please include the image

File: docs/mevblocker/README.mdx [hunk 2]
```diff
@@ -1,4 +1,40 @@
...
+
+  - **Faster Inclusion over other RPCs**: Instead of sending your transaction to the Ethereum public mempool with a low tip and waiting for it to be included, you send your transaction to MEV Blocker private pool which forwards your transactions directly to all major builders. This yields a faster inclusion rate as your transaction is already on the builders’ radar and will likely be included in the current or next block available.
+
+  - **Multiple Endpoints**: MEV Blocker provides three different RPC endpoints for you to choose based on your transaction needs. Learn more about each endpoint’s trade-offs here. 
+
+  - **Transparency**: MEV Blocker has a [Dune Dashboard](https://dune.com/cowprotocol/mev-blocker)
+where you can check all your transactions submitted through the MEV Blocker RPC. You can also follow your transactions  in detail, see all searcher bids, and calculate the overall revenue you got via MEV Blocker rebates.
+
+IMG
```
Comment (mfw78) [inline docs/mevblocker/README.mdx:24]: Please include the image

File: docs/mevblocker/README.mdx [hunk 3]
```diff
@@ -1,4 +1,40 @@
...
+
+Start sending transactions today using [MEV Blocker Endpoint](https://rpc.mevblocker.io). Feel free to send transactions directly or reach out for support.
+
+## For Searchers
+
+You can already bid on auctions today through MEV Blocker’s permissionless Order Flow Auction system. No KYC required.
+
+## For Builders
+
+In order to receive MEV Blocker Order Flow, your builder must meet certain requirements. You can find more information about the requirements here.
```
Comment (mfw78) [inline docs/mevblocker/README.mdx:40]: Is there meant to be a link here?

**Review by mfw78 — COMMENTED** [review]
Please ensure only typewriter quotes and apostrophes (ie. " and ', not “/”/’)

File: docs/mevblocker/README.mdx (hunk 1 above)
Comment (cowmarketing) [reply docs/mevblocker/README.mdx:5]: Well, I havent finished it yet, was just testing it

File: docs/mevblocker/README.mdx (hunk 3 above)
Comment (cowmarketing) [reply docs/mevblocker/README.mdx:40]: Not yet.

**Comment by cowmarketing** [comment]
I agree to the CoW Swap CLA

**Comment by cowmarketing** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by mfw78** [comment]
This has been superseded by #352

**Pull request closed without merging**